tests/e2e/support/objects/app-files/spaces/actions.ts

```diff
@@ -88,6 +88,7 @@ export interface openSpaceArgs {
 export const openSpace = async (args: openSpaceArgs): Promise<void> => {
   const { page, id } = args
   await page.locator(util.format(spaceIdSelector, id)).click()
+  await page.locator('.space-header').waitFor()
```

Comment:
```suggestion
  await page.locator('<selectorVar>').waitFor()
```
IMO it's better to use a variable for the selector.